livesync_citadel/indico_livesync_citadel/plugin.py

```diff
@@ -0,0 +1,44 @@
+# This file is part of the Indico plugins.
+# Copyright (C) 2002 - 2020 CERN
+#
+# The Indico plugins are free software; you can redistribute
+# them and/or modify them under the terms of the MIT License;
+# see the LICENSE file for more details.
+
+from __future__ import unicode_literals
+
+from wtforms.fields.core import StringField
+from wtforms.fields.html5 import URLField
+from wtforms.validators import URL, DataRequired
+
+from indico.web.forms.base import IndicoForm
+
+from indico_livesync import LiveSyncPluginBase
+from indico_livesync_citadel import _
+from indico_livesync_citadel.backend import LiveSyncCitadelBackend
+from indico_livesync_citadel.blueprint import blueprint
+
+
+class LiveSyncCitadelSettingsForm(IndicoForm):
+    search_backend_url = URLField(_('Search backend URL'), [DataRequired(), URL()],
+                                  description=_('Search backend host:port'))
+    search_backend_token = StringField(_('Search backend token'), [DataRequired()],
+                                       description=_('Authentication token for the Search backend'))
+    tika_server = URLField(_('Tika server URL'), [DataRequired(), URL()],
+                           description=_('Tika server host:port. '
```

Comment:
Is there any authentication? Also, if this expects a proper URL then the `host:port` is a bit misleading...

Reply:
> Is there any authentication?

Not sure about this. I will discuss it with the search developers.